src/estimagic/visualization/colors.py

```diff
@@ -1,266 +1,44 @@
-import numpy as np
-from estimagic.config import IS_MATPLOTLIB_INSTALLED
-
-if IS_MATPLOTLIB_INSTALLED:
-    from matplotlib.colors import LinearSegmentedColormap
-
-
-def get_colors(palette, number, as_cmap=False, skip_dark=0, skip_bright=0):
+def get_colors(palette, number):
     """Return a list with hex codes representing a color palette.
 
     Args:
-        palette (str): One of ["categorical", "ordered", "blue", "red", "yellow",
-            "green", "orange", "purple"] or combinations of two colors, e.g.
-            "red-green".
-        number (int): Number of colors needed.
-            - For combined color palettes like (red-blue) it is between 0 and 24
-            - For non-combined ordered palettes like blue it is between 0 and 12
-            - For the categorical palette it can by any non-negative integer. The colors
-              are repeated if more than 12 colors are requested.
-        as_cmap (bool): If True, the result is returned as matplotlib cmap.
-        skip_dark (int): How many colors to skip from the dark side. Only
-            available for monochrome and combined color palettes.
-        skip_bright (int): How many colors to skip from the bright side. Only
-            available for monochrome and combined color palettes.
+        palette (str): One of ["categorical", "ordered"]
+        number (int): Number of colors needed. Colors are repeated if more than eight
```

Comment:
This should only be for categorical. Repeating ordered should probably rather interpolate a cmap somehow.

Reply:
We only use the categorical color scale in estimagic at the moment. However, I think it makes sense to support ordered colors for the future. One option would be to switch to the `nipy_spectral` cmap once we exceed the 6 ordered colors we have: 

![image](https://user-images.githubusercontent.com/9567321/140906259-373d2985-81c5-409a-af70-b0e0ca1c718e.png)

Reply:
Good solution.